prov/efa/src/rdm/rxr_pkt_type_base.c

```diff
@@ -524,27 +462,26 @@ int rxr_pkt_copy_data_to_cuda(struct efa_rdm_ep *ep,
  * When application's receive buffer is on host, data is copied immediately, and
  * rxr_pkt_handle_copied() is called.
  *
- * @param[in]		ep		endpoint
+ *
+ * @param[in]		pke	the packet entry that contains data
  * @param[in,out]	ope	ope contains information of the receive
- *                      	        op. This function uses receive buffer in it.
- * @param[in]		data_offset	data offset in the packet in the receiving buffer.
- * @param[in]		pkt_entry	the packet entry that contains data
- * @param[in]		data		the pointer pointing to the beginning of data
- * @param[in]		data_size	the length of data
+ *                      	buffer.
  * @return		On success, return 0
  * 			On failure, return libfabric error code
  */
-ssize_t rxr_pkt_copy_data_to_ope(struct efa_rdm_ep *ep,
-				      struct efa_rdm_ope *ope,
-				      size_t data_offset,
-				      struct efa_rdm_pke *pkt_entry,
-				      char *data, size_t data_size)
+ssize_t efa_rdm_pke_copy_payload(struct efa_rdm_pke *pke,
```

Comment:
can we name it explicitly as `copy_payload_to_ope`

Reply:
I can. but techniquely, we are not copy data to ope, but copy data to user buffer. ope has the information of the user buffer,

Reply:
I think I will name it `copy_payload_to_ope`, and rename the other funciton "init_payload_from_ope"

Reply:
I have an commit that merge this file into efa_rdm_pke_utils.c, can I do the renaming in that commit?

Reply:
Sure